lib/sidekiq_simple_delay/delay_worker.rb

```diff
@@ -22,8 +22,8 @@ def perform(args)
           target_klass
         end
 
-      method_name = args['m']
-      method_args = args['m_args']
+      method_name = args['method']
+      method_args = args['method_args']
```

Comment:
Since these are part of the "internal api" I'd like to keep them terse to reduce memory pressure on redis. Every little bit helps.